docker/BuildAndroid.dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:noble
+WORKDIR /usr/local/app
+
+RUN apt-get update && apt-get install -y openjdk-17-jdk python3 pipx cmake ccache ninja-build wget git xz-utils
+
+ENV PIPX_HOME="/opt/pipx"
+ENV PIPX_BIN_DIR="/usr/local/bin"
+ENV PIPX_MAN_DIR="/usr/local/share/man"
+RUN pipx install 'conan<2.0'
+RUN pipx install 'sdkmanager==0.6.10'
+
+RUN conan profile new conan --detect
+
+RUN wget https://github.com/vcmi/vcmi-dependencies/releases/download/1.3/dependencies-android-arm64-v8a.txz
+RUN tar -xf dependencies-android-arm64-v8a.txz -C ~/.conan
+RUN rm dependencies-android-arm64-v8a.txz
+
+ENV JAVA_HOME="/usr/lib/jvm/java-17-openjdk-amd64"
+ENV ANDROID_HOME="/usr/lib/android-sdk"
+ENV GRADLE_USER_HOME="/vcmi/.cache/grandle"
+ENV GENERATE_ONLY_BUILT_CONFIG=1
+
+RUN sdkmanager --install "platform-tools"
+RUN sdkmanager --install "platforms;android-34"
+RUN yes | sdkmanager --licenses
+
+RUN conan download android-ndk/r25c@:4db1be536558d833e52e862fd84d64d75c2b3656 -r conancenter
```

Comment:
is it not downloaded automatically when using profile with NDK?

Reply:
Yes. But it's also here because this step is better to do in image building part, not in executing script.